lib/DDG/Spice/IndegoBikeshareLocations.pm

```diff
@@ -0,0 +1,34 @@
+package DDG::Spice::IndegoBikeshareLocations;
```

Comment:
On second thought, let's go with the approach @marianosimone already has. `DDG::Spice::BikeSharing::IndegoPHL`